Punky Brewster/BeerListTableViewController.swift

```diff
@@ -25,21 +25,21 @@ class BeerListTableViewController: UITableViewController {
         let request = BeerListRequest()
         request.perform {
             (retrievedBeers:[Beer], error:NSError?) -> Void in
-            
-            if(error != nil) {
-                print("Error: \(error)")
-            } else {
-                self.beers = retrievedBeers
-            }
-            
+ 
             dispatch_async(dispatch_get_main_queue(), {
                 if sender != nil {
                     sender?.endRefreshing()
                 } else {
                     self.activityIndicator.stopAnimating()
                 }
-
-                self.tableView.reloadData()
+ 
+                if error != nil {
+                    self.performSegueWithIdentifier("showError", sender: self)
+                    print("Request failed with status code: \(error!.userInfo["statusCode"])")
```

Comment:
This line failed to compile for me. Given it seems to be a debug statement, shall we just kill it?